packages/components/react-web/.storybook/theme.js

```diff
@@ -4,13 +4,13 @@ import * as elements from '@compassion-gds/elements';
 export default create({
   base: 'light',
 
-  colorPrimary: elements.color.primary,
+  colorPrimary: elements.color.primary.base,
```

Comment:
There should be no `.base` color, we decided to use `.main`

Reply:
Fixed!